addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -132,6 +132,13 @@ def test_alias_setup(self):
         with self.assertRaises(exceptions.ValidationError):
             alias.write({'alias_defaults': "{'custom_field': brokendict"})
 
+    def test_alias_sanitize(self):
```

Comment:
Technically should be located before ``test_alias_setup`` (as tests are called alphabetically, it eases reading logs and finding the test when having multiple crashing tests).

I know test_alias_name_unique is badly located but hey.